src/panels/profile/ha-mfa-modules-card.js

```diff
@@ -0,0 +1,127 @@
+
+import EventsMixin from '../../mixins/events-mixin.js';
+import LocalizeMixin from '../../mixins/localize-mixin.js';
+
+let registeredDialog = false;
+
+/*
+ * @appliesMixin EventsMixin
+ * @appliesMixin LocalizeMixin
+ */
+class HaMfaModulesCard extends EventsMixin(LocalizeMixin(PolymerElement)) {
+  static get template() {
+    return html`
+    <style include="iron-flex ha-style">
+      .error {
+        color: red;
+      }
+      .status {
+        color: var(--primary-color);
+      }
+      .error, .status {
+        position: absolute;
+        top: -4px;
+      }
+      paper-card {
+        display: block;
+        max-width: 600px;
+        margin: 16px auto;
+      }
+      paper-button {
+        color: var(--primary-color);
+        font-weight: 500;
+        margin-right: -.57em;
+      }
+    </style>
+    <paper-card heading="Multi-factor Authenication Modules">
+      <template is="dom-repeat" items="[[mfaModules]]" as="module">
+        <paper-item>
+          <paper-item-body two-line="">
+            <div>[[module.name]]</div>
+            <div secondary="">[[module.id]]</div>
+          </paper-item-body>
+          <template is="dom-if" if="[[module.enabled]]">
+            <paper-button on-click="_disable">Disable</paper-button>
+          </template>
+          <template is="dom-if" if="[[!module.enabled]]">
+            <paper-button on-click="_enable">Enable</paper-button>
+          </template>
+        </paper-item>
+      </template>
+    </paper-card>
+`;
+  }
+
+  static get properties() {
+    return {
+      hass: Object,
+
+      _loading: {
+        type: Boolean,
+        value: false,
+      },
+
+      // Error message when can't talk to server etc
+      _statusMsg: String,
+      _errorMsg: String,
+
+      mfaModules: Array,
+    };
+  }
+
+  connectedCallback() {
+    super.connectedCallback();
+
+    if (!registeredDialog) {
+      registeredDialog = true;
+      this.fire('register-dialog', {
+        dialogShowEvent: 'show-mfa-module-setup-flow',
+        dialogTag: 'ha-mfa-module-setup-flow',
+        dialogImport: () => import('./ha-mfa-module-setup-flow.js'),
+      });
+    }
+  }
+
+  disconnectedCallback() {
+    super.disconnectedCallback();
+    if (this._unsubEvents) this._unsubEvents();
+  }
+
+  _enable(ev) {
+    this.fire('show-mfa-module-setup-flow', {
+      hass: this.hass,
+      mfaModuleId: ev.model.module.id,
+      dialogClosedCallback: () => this._refreshCurrentUser(),
+    });
+  }
+
+  _disable(ev) {
+    if (!confirm(`Are you sure you want to disable ${ev.model.module.name}?`)) return;
+
+    const mfaModuleId = ev.model.module.id;
+
+    this.hass.callWS({
+      type: 'auth/depose_mfa',
+      mfa_module_id: mfaModuleId,
+    }).then(() => {
+      this._refreshCurrentUser();
+    });
+  }
+
+  async _refreshCurrentUser() {
+    const user = await this.hass.callWS({
+      type: 'auth/current_user',
+    });
+    this.mfaModules = user.mfa_modules;
```

Comment:
This is a property that is passed in. So you will need to tell the `hass` object to update the user by firing an event.  Add support for this event [here](https://github.com/home-assistant/home-assistant-polymer/blob/master/src/layouts/app/auth-mixin.js)